addons/website_sale/models/product_template.py

```diff
@@ -58,6 +58,8 @@ class ProductTemplate(models.Model):
         digits='Product Price',
         help="The amount will be displayed strikethroughed on the eCommerce product page")
 
+    product_error_message = fields.Char(compute='_compute_product_error_message', string='Error Message')
```

Comment:
Why is this field needed? Where is the `_compute_product_error_message`?
I saw in your other PR (https://github.com/odoo/enterprise/pull/35676) that the field is only used in `_compute_product_error_message`, which calls a super that doesn't even exist.

Reply:
Hi, thanks for the review, I was still working with the issue since the previous fix wasn't a good approach for stable, I got a different solution, It is simple, If I can get a review (in the enterprise PR) so I can close this one If It's enough with that fix 🙂